src/modules/library.ts

```diff
@@ -129,7 +129,7 @@ export class Library {
   /** Verifies that the Focused Element returned from {@link getFocusedElement} is of a certain type (XMLName/tag) */
   async verifyFocusedElementIsOfCertainTag(tag: string, maxRetries = 10) {
     let retries = 0;
-    let element: elementValueParsed;
+    let element: elementValueParsed = { XMLName: undefined, Attrs: {} };
```

Comment:
Does this work even if there is a `Nodes` key that gets added?

Reply:
yeah it will be overwritten, just avoids the undefined exception in the while loop, was trying to access XMLName while elementValueParsed was undefined